### ISSUE TITLE
Debug & variable inspector break if filename case in editor differs to project explorer

### ISSUE THREAD

**Issue by bengray2** [microsoft/vscode#100924]
If the filename case differs in the code editor to that of the project explorer, the debug breakpoints & variable inspector do not work properly.  When the debug breakpoint is reached code execution stops, but the line is not highlighted and the variable inspector does not appear when hovering the mouse over variables.

- VSCode Version: 1.46.1
- OS Version: Windows 10

Steps to Reproduce:

1.  Create a C# project (dotnet new webapi)
2.  Create a class called ElasticSearchHelper.cs
3.  Add a method and some code to the class
4.  Place a breakpoint over a variable and debug (the breakpoint line is highlighted, and variable can be inspected by mouse hover)
5.  Stop debugging
6.  Leave the code editor open  (this is important)
7.  Rename the file to ElasticsearchHelper.cs  (changing the case of 's' in Search from capital to lower case)
8.  Debug again - the code execution stops at the breakpoint, but the line is not highlighted
9.  Hover the mouse over variables - the variables value does not appear

Does this issue occur when all extensions are disabled?: Yes

**Comment by bengray2** [comment]
I forgot to add, you can easily work around this by closing the editor window and re-opening, then it will use the rename.  But it drove me mad figuring out what was happening and restarting VSCode doesn't fix the issue because the tab with the incorrect filename stays open.

**Comment by weinand** [comment]
@isidorn it should now be possible to make this case work.

**Comment by ilovett** [comment]
Yea I ran into this issue for the first time yesterday, I renamed a file from `thing.ts` to `Thing.ts`.  I was confused for a bit why my breakpoints were not mapping in that file only.  Eventually I noticed that the tab said `thing.ts` and sidenav file said `Thing.ts`.  I tried closing all tabs, clearing recently opened, search history, editor history, etc., then reopening.  Cannot get the file to open in the tab as `Thing.ts` (new case filename).  Even renaming to something else, `derp.ts` and back to `Thing.ts` would not fix the filename in the tab.

Finally, opening a new vscode window and starting a new project would "fix" it.  However now my project will always open the file as the original case sensitive `thing.ts` and I cannot reset or fix the tab to open as `Thing.ts` which is pretty annoying.
